dask_sql/physical/utils/filter.py

```diff
@@ -6,6 +6,7 @@
 
 import dask.dataframe as dd
 import numpy as np
+import pandas as pd
 from dask.blockwise import Blockwise
 from dask.highlevelgraph import HighLevelGraph, MaterializedLayer
 from dask.layers import DataFrameIOLayer
```

Comment:
Need to do a `from dask_sql.mappings import parse_datetime` below `from dask_sql._compat import PQ_IS_SUPPORT, PQ_NOT_IN_SUPPORT`.